### PR TITLE
Practise lesson three

### PR DESCRIPTION
**Pull request by KudAndrii**

### CODE REVIEW

**Comment by KudAndrii** [comment]
some comment

**Comment by DmytroZhuravlov14** [comment]
i dont like

File: PractiseLessonThree/PractiseLessonThree/Program.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+namespace PractiseLessonThree
+{
+    /// <summary>
+    /// Point of Entering the program.
+    /// </summary>
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            string input = string.Empty;
```
Comment (DmytroZhuravlov14) [inline PractiseLessonThree/PractiseLessonThree/Program.cs:14]: not string empty

**Review by DmytroZhuravlov14 — CHANGES_REQUESTED** [review]

File: PractiseLessonThree/PractiseLessonThree/Program.cs (hunk 1 above)
Comment (KudAndrii) [reply PractiseLessonThree/PractiseLessonThree/Program.cs]: fixed

**Pull request merged**